README.md

```diff
@@ -43,9 +43,11 @@ The `Client` wraps a single HTTP connection to a specified ElasticSearch host/po
 (At present there is no connection pooling, each client has one connection; if you need multiple connections you will need multiple clients.  This may change in the future).
 
 ```rust
-let mut client = Client::new("localhost", 9200);
+let mut client = Client::new("http://localhost");
```

Comment:
Could we update the URL to `http://localhost:9200` as I think that better shows the intent.